app/developers/templates/developers/manage_question_condition_select_condition_type.html

```diff
@@ -0,0 +1,72 @@
+{% extends "deliver_grant_funding/grant_base.html" %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+{% from "govuk_frontend_jinja/components/radios/macro.html" import govukRadios %}
+{% from "govuk_frontend_jinja/components/input/macro.html" import govukInput %}
+{% from "govuk_frontend_jinja/components/button/macro.html" import govukButton %}
+{% from "govuk_frontend_jinja/components/checkboxes/macro.html" import govukCheckboxes %}
+{% from "govuk_frontend_jinja/components/summary-list/macro.html" import govukSummaryList %}
+{% from "govuk_frontend_jinja/components/notification-banner/macro.html" import govukNotificationBanner %}
+{% from "developers/_number_expression.html" import render_number_expression %}
+
+{% block pageTitle %}
+  Add a condition - {{ question.name }} - MHCLG Funding Service
+{% endblock pageTitle %}
+
+{% set active_item_identifier = "developers" %}
+
+{% set is_editing = expression %}
+{% set heading = "Manage condition" if is_editing else "Add a condition" %}
```

Comment:
The language for validations is `Edit validation` - happy to use Manage but if you can make it consistent one way or the other please 🙏

Reply:
Nope I think I just called it this because it was the same as the template/ method names we were using? 

Edit is all good I'll clean that up